src/entity/posts/Post.ts

```diff
@@ -101,7 +101,6 @@ export class Post {
   metadataChangedAt: Date;
 
   @Column({ type: 'text', default: UNKNOWN_SOURCE })
-  @Index()
```

Comment:
Curious how this does not get hit for example in `sourceFeed` where we have `builder.andWhere('s.sourceId = :sourceId', { sourceId });`

Reply:
Not actually sure tbh, anyway we can verify it further? (maybe it was a duplicate?)

Reply:
Maybe quickly try with EXPLAIN ANALYSE and see if there are differences for those queries locally with or without index? Just to validate, we might get some good insight.

Reply:
@capJavert :

With index on source_id:
![Screenshot 2024-02-15 at 15 58 50](https://github.com/dailydotdev/daily-api/assets/554874/8a8d2b6b-c715-4bb0-ac83-af846856158f)

Without index on source_id:
![Screenshot 2024-02-15 at 15 59 20](https://github.com/dailydotdev/daily-api/assets/554874/5db3ab68-32ff-4ea9-9181-13409c0b6644)

Reply:
Ah yes, so it reuses `IDX_post_source_id_pinned_at_created_at` if the column index is not there. So basically what could happen is that we remove `IDX_post_source_id_pinned_at_created_at` and then you end up with slow queries.

Reply:
it does not have to use specific index. if it constantly has some index in memory (because it's used for other things) - it will stick to using it even if there is potentially more "narrow" alternative index.

Reply:
Ah didn't realize that's how it would prefer indexes, always thought it would go for the narrowest one.
Anyway we have a tool to tell us if we MISS indexes, not the other way around, so let's just remove it for now 😅